### PR TITLE
Add legal page

### PR DESCRIPTION
**Pull request by jshorty**
Just a bit of brute force copy pasting into headings and paragraphs. Preserves the bold and italicized content from the .docx reference file— I have no idea what's standard in this world...

### CODE REVIEW

File: legal.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+permalink: /legal
+layout: front
+---
+
+<header class="masthead text-center">
+  <a class="js-scroll-trigger" href="/#page-top"><img class="logo-apply" src="../img/theplatefund_logo.png"></a>
+</header>
```
Comment (derekslager) [inline legal.html:8]: I think we have this in an include somewhere -- or at least please use one of the updated versions so it works on the translated site.

Comment (derekslager) [reply legal.html:8]: This is related to item #10 on the issue tracker I think.

Comment (jshorty) [reply legal.html]: My bad, missed the changes— pushed it into one HTML file that's then only referenced in our sole layout file

File: _includes/logo_header.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<header class="masthead text-center">
+  <a class="js-scroll-trigger" href="{{ site.baseurl }}/"><img class="logo-apply" src="{{ "/img/theplatefund_logo.png" | prepend: site.baseurl_root }}"></a>
+  <h2 class="mt-5">{% translate apply.title %}</h2>
```
Comment (derekslager) [inline _includes/logo_header.html:3]: I think this is the header for the apply page, assuming this makes it show up everywhere?

Comment (derekslager) [reply _includes/logo_header.html:3]: Seems so, I assume the layout will still work if you pull that out of the header on the apply page.

Comment (jshorty) [reply _includes/logo_header.html]: good catch, fixed as you describe

**Pull request merged**